components/BottomNavigation/src/MDCBottomNavigationBar.h

```diff
@@ -151,6 +151,18 @@ typedef NS_ENUM(NSInteger, MDCBottomNavigationBarAlignment) {
  */
 @property(nonatomic, assign) CGFloat itemsContentHorizontalMargin;
 
+/**
+ Flag to allow clients to gradually correct the size/position of the Bottom Navigation bar relative
+ to the safe area on iOS 11+.
+
+ NOTE: In an upcoming release, this flag will be removed and the default behavior will be to exclude
+ the safe area in size calculations.
+
+ Defaults to @c NO.
+ */
+@property(nonatomic, assign) BOOL sizeThatFitsIncludesSafeArea __deprecated_msg(
+    "Safe area should be determined by superviews and added to the bar's bounds.");
```

Comment:
To confirm my understanding: if the client does not change this property does this PR change current behavior?

If so, and if we are concerned about the possibility of this behavior breaking existing clients, then please follow this flow:

1. Set the property to the legacy behavior.
2. Land the code in a release.
3. Follow the deprecation policy to deprecate and delete the behavior toggle, including writing a migration guide that encourages clients to enable the new behavior: https://github.com/material-components/material-components-ios/blob/develop/contributing/deprecation_policy.md

Reply:
Your understanding is correct. Since it is a bug, I didn't originally plan on letting clients migrate gradually - I expected to fix internal clients immediately with a patched change for the Release. I've updated this PR to change the behavior of the flag and add migration documentation. Please take a look.